### DISCUSSION TITLE
Could pinch to zoom wrap the text round the screen?

### DISCUSSION THREAD

**Discussion by SmutnyStefan** [tughi/aggregator-android#54] [Ideas]
I like the feature, but zooming could work better by wraping the text instead of keeping it fixed

**Comment by tughi** [comment]
Actually the zooming should have been disabled... 

What you want is an option to increase the text size, right?
This isn't supported yet.

**Comment by SmutnyStefan** [comment]
I liked this since I am short sighted and not many feed readers gives a f*k about such users. But then again I have other tools to deal with my issues. It would be nice to have it, but it is not critical imho

**Reply by tughi** [reply]
The problem with pinch-to-zoom is that it interferes with the swipe gesture
